web/packages/shared/components/MenuLogin/MenuLogin.tsx

```diff
@@ -259,6 +298,12 @@ const StyledMenuItem = styled(MenuItem)(
   font-size: 12px;
   border-bottom: 1px solid ${theme.colors.spotBackground[0]};
   min-height: 32px;
+  display: inline-block;
+  text-align: left;
+  max-width: 450px;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
```

Comment:
The menu theoretically looks good if we completely remove all of these styles, so let's add a comment explaining why they're needed.